### PR TITLE
File Splitting for proxy and utils

### PR DESCRIPTION
**Pull request by Risewill23**

### CODE REVIEW

**Comment by Risewill23** [comment]
Why is saying it isn't black formatted?

**Comment by ixi2101** [comment]
> Why is saying it isn't black formatted?

It's failing on the linting not black
Run 'nox' in the directory and it will list a load of issues that need remedying such as unused imports

**Comment by Risewill23** [comment]
I finally fixed everything

**Comment by thoroc** [comment]
Would it be a pain to ask the methods name to be snake case?

**Comment by Risewill23** [comment]
Fixed

**Comment by thoroc** [comment]
Have the method calls been updated? We don't want to break the script in such a trivial manner XD

**Comment by Risewill23** [comment]
Yes I did find and replace for all of main.py

File: main.py [hunk 1]
```diff
@@ -265,7 +109,7 @@ def set_pixel_and_check_ratelimit(
         }
 
         response = requests.request(
-            "POST", url, headers=headers, data=payload, proxies=self.GetRandomProxy()
+            "POST", url, headers=headers, data=payload, proxies=proxy.GetRandomProxy()
```
Comment (thoroc) [inline main.py:112]: Does this method call not need to be updated?

Comment (Risewill23) [reply main.py:112]: Oh wait

**Comment by Risewill23** [comment]
Now I'm finally finally done

**Comment by JAKONL** [comment]
I have looked it over and it looks good. I cloned it and ran it with 2 accounts for 5 minutes. Seems to work just fine. Unfortunately I cannot test the tor or proxy support at the moment.

**Review by JAKONL — APPROVED** [review]

**Comment by JAKONL** [comment]
Tested tor functionality successfully. Everything seems to be working as expected. Cheers @Risewill23

**Pull request merged**